test/e2e/deploy_reaper_test.go

```diff
@@ -93,6 +99,14 @@ var _ = Describe("Deploy Reaper with Cassandra backend", func() {
 				return len(reaper.Status.Clusters) == 1 && reaper.Status.Clusters[0] == cassdc.Spec.ClusterName
 			})
 			Expect(err).ToNot(HaveOccurred(), "failing waiting for cluster to get registered")
+
+			By("verify from reaper that the cluster has scheduling")
```

Comment:
I do think it makes sense to have separate e2e tests where auto scheduling is enabled and not enabled.